features/admin.identity-providers.v1/utils/identity-provider-management-utils.ts

```diff
@@ -136,6 +136,29 @@ export class IdentityProviderManagementUtils {
             return authenticator?.tags?.includes("Custom");
         };
 
+        /**
+         * Resolves the image of the local authenticator.
+         *
+         * For system defined user authenticators, the function retrieves the default icon.
+         * For custom local authenticators, it returns the authenticator's own image URI if available,
+         * or falls back to a default custom authenticator icon.
+         *
+         * @param authenticator - Authenticator.
+         * @returns image URI of the authenticator.
+         */
+        const resolveLocalAuthenticatorImage = (authenticator: LocalAuthenticatorInterface): string => {
+            if (!isCustomLocalAuthenticator(authenticator)) {
+                AuthenticatorMeta.getAuthenticatorIcon(authenticator.id);
+            }
+
+            // Resolve image URI of custom local authenticators.
+            if (authenticator?.image) {
+                return authenticator.image;
+            } else {
+                AuthenticatorMeta.getCustomAuthenticatorIcon();
+            }
```

Comment:
Lets add the icon resolving logic to `AuthenticatorMeta.getCustomAuthenticatorIcon()` method